### PR TITLE
Use trackalert  to create SETKEY

### PR DESCRIPTION
**Pull request by bgmex**
Minor fix

### CODE REVIEW

**Comment by neilcook** [comment]
So, the -c flag makes trackalert run as a client, connecting to a server. This will not generate a key, it will simply fail to connect with an error message. I'm not sure what the reason for this PR is? Is the current post install script not working?

**Comment by bgmex** [comment]
I'm sorry for my poor PR. I forgot to add the explanation.

Meanwhile I did some further research and learned, that my PR is insufficient/unnecessary.

The current post install script for wforce-trackalert only fails if wforce is already running. My PR fixes this.
If wforce is not running my PR fails and the current post install scripts works.

A proper fix would be something like:
```
systemctl is-active --quiet wforce
if [ $? -eq 0 ]; then
    # wforce is running so connect to it using -c
    SETKEY=`echo "makeKey()" | wforce -c | grep setKey`
else
    # wforce is not running
    SETKEY=`echo "makeKey()" | wforce | grep setKey`
fi
```
Or we that for granted, that wforce is not running when wforce-trackalert is being installed

**Comment by neilcook** [comment]
Actually the proper fix is to run trackalert to generate the key not wforce, as trackalert should have no dependency on wforce being installed:
```
SETKEY=`echo "makeKey()" | trackalert | grep setKey`

**Comment by neilcook** [comment]
Note that this issue also exists in the spec file for RH packages

File: builder-support/debian/wforce-trackalert.postinst [hunk 1]
```diff
@@ -17,7 +17,7 @@ case "$1" in
       echo "done"
     fi
     echo -n "Modifying trackalert.conf to replace password and key..."
-    SETKEY=`echo "makeKey()" | wforce | grep setKey`
+    SETKEY=`echo "makeKey()" | wforce -c | grep setKey`
```
Comment (neilcook) [inline builder-support/debian/wforce-trackalert.postinst:20]:
```suggestion
    SETKEY=`echo "makeKey()" | trackalert -c | grep setKey`
```

Comment (bgmex) [reply builder-support/debian/wforce-trackalert.postinst:20]: ```trackalert -c``` needs trackalert.service to be running. While installing trackalert we can assume it's not running
```suggestion
    SETKEY=`echo "makeKey()" | trackalert | grep setKey`
```

Comment (neilcook) [reply builder-support/debian/wforce-trackalert.postinst:20]: yes this change would work

**Review by neilcook — APPROVED** [review]

**Pull request merged**